packages/dashmate/src/listr/tasks/setup/regular/registerMasternodeGuideTaskFactory.js

```diff
@@ -62,8 +63,8 @@ function registerMasternodeGuideTaskFactory() {
               message: 'Which wallet will you use to store keys for your masternode?',
               choices: [
                 { name: REGISTRARS.CORE, message: 'Dash Core Wallet' },
-                { name: REGISTRARS.ANDROID, message: 'Android Dash Wallet' },
-                { name: REGISTRARS.IOS, message: 'iOS Dash Wallet' },
+                // { name: REGISTRARS.ANDROID, message: 'Android Dash Wallet' },
+                // { name: REGISTRARS.IOS, message: 'iOS Dash Wallet' },
                 { name: REGISTRARS.OTHER, message: 'Other' },
```

Comment:
Are there any differences between `Dash Core Wallet` and `Other` register methods?

Reply:
Nope :)

Reply:
Yeah, looks like. We can always enable it back, when we implement something

Reply:
It seems DMT integration is on going. I wouldn't disable prompt atm to enable it back in a week.